### PR TITLE
[FEAT[ - Add multivariate evaluation to CI

### PR DESCRIPTION
**Pull request by marcopeix**
Add a step to evaluate multivariate forecasting models in the CI. Right now, it uses the ETTm2 dataset with a horizon of 96. Models tested are: TSMixer, SOFTS, iTransformer and AutoiTransformer. More models can be added.

### CODE REVIEW

**Review by elephaint — CHANGES_REQUESTED** [review]
I think for the first iteration would be good to include all multivariate models, i.e. also StemGNN, MLPMultivariate and TSMixerx, and their Auto counterparts. Wdyt? 

I know the univariate testing is also incomplete, so not sure what the strategy is on that. I think we should test all models minimally with the CI on every change.

File: action_files/test_models/src/multivariate_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    evals = pd.concat([evals, times], axis=1)
+
+    return evals
+
+
+if __name__ == '__main__':
+
+    groups = ['ETTm2']
+    
+    models = ['SOFTS',
```
Comment (elephaint) [inline action_files/test_models/src/multivariate_evaluation.py:46]: I'd add all multivariate models here

File: action_files/test_models/src/multivariate_models.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+from src.data import get_data
+
+os.environ['NIXTLA_ID_AS_COL'] = '1'
+
+
+def main(dataset: str = 'multivariate', group: str = 'ETTm2') -> None:
+    train, horizon, freq = get_data('data/', dataset, group)
+    train['ds'] = pd.to_datetime(train['ds'])
+
+    models = [
```
Comment (elephaint) [inline action_files/test_models/src/multivariate_models.py:27]: I'd add all multivariate models here

**Review by elephaint — CHANGES_REQUESTED** [review]
Nice!

File: action_files/test_models/src/multivariate_evaluation.py (hunk 1 above)
Comment (marcopeix) [reply action_files/test_models/src/multivariate_evaluation.py:46]: What model am I missing? I'm only including models inheriting BaseMultivariate.

File: action_files/test_models/src/multivariate_models.py (hunk 2 above)
Comment (marcopeix) [reply action_files/test_models/src/multivariate_models.py:27]: Same here, not sure which model I'm missing. We have SOFTS, TSMixer/TSMixerx, iTransformer, MLPmultivariate, StemGNN. Missing TimeMixer, but that PR is not merged yet.

**Review by elephaint — APPROVED** [review]

**Pull request merged**